pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableBuilder.java

```diff
@@ -77,6 +77,9 @@
 // TODO:   3. Given a data schema, write all values one by one instead of using rowId and colId to position (save time).
 // TODO:   4. Store bytes as variable size data instead of String
 public class DataTableBuilder {
+  public static final int VERSION_2 = 2;
+  public static final int VERSION_3 = 3;
+  private static int _version = VERSION_3;
```

Comment:
This should not be hardcoded but from the config

Reply:
We have a `setCurrentDataTableVersion` static function to set versions, which is called in `HelixServerStarter`